src/cromshell/utilities/submissions_file_utils.py

```diff
@@ -73,3 +75,39 @@ def update_submission_db(submission_file_path: Union[str, Path]) -> bool:
         )
 
     return old_format  # for tests
+
+
+def update_row_values_in_submission_db(
+    workflow_database_path: str,
+    workflow_id: str,
+    column_to_update: str,
+    update_value: str,
+) -> None:
+    """
+    Updates the all_workflow_database_tsv for a given workflow_id and column
+    :param workflow_database_path: Path to all_workflow_database tsv file
+    :param workflow_id: Hexadecimal identifier of workflow submission
+    :param column_to_update:["STATUS", "ALIAS"]
+    :param update_value: Value of the cell to update
+    :return:
+    """
+
+    available_columns = [column.value for column in MutableSubmissionFileHeader]
+    if column_to_update not in available_columns:
+        raise ValueError(
+            f"Invalid column_to_update: '{column_to_update}'. "
+            f"Expected one of: '{available_columns}'"
+        )
+
+    # Update config.submission_file:
+    with fileinput.FileInput(
+        workflow_database_path, inplace=True, backup=".bak"
+    ) as csv_file:
+        reader = csv.DictReader(csv_file, delimiter="\t")
+        print("\t".join(reader.fieldnames))  # print statement rewrites file header
+        for row in reader:
+            if row["RUN_ID"] == workflow_id:
```

Comment:
While reusing this function for something else, I found out that if for whatever reason it stumbles over one of the parameters (eg. column name ) it will silently fail and leave with a half-completed file. I'll rewrite it so that it edits a temp file first then writes the original with the old. that way if it fails the original will still be unedited